### DISCUSSION TITLE
[Feature Request] Notion like Paste as

### DISCUSSION THREAD

**Discussion by hmasum52** [docmost/docmost#1626] [Ideas]
Notion like `paste as`
![image](https://github.com/user-attachments/assets/97f1a061-8b27-40cb-8f04-55e3a2cc4811)

paste as `mention`
![image](https://github.com/user-attachments/assets/bf82f734-360d-4ec3-b389-7bd0ed34c320)

paste as `bookmark`
![image](https://github.com/user-attachments/assets/e3792168-1b3e-4a17-84dd-fac5e6f685eb)

paste as `embedded`
![image](https://github.com/user-attachments/assets/030a1e26-4396-449c-a45c-7c9804fc8081)

**Comment by hmasum52** [comment]
I will open a PR

**Comment by SyndicatedPillbug** [comment]
This would be awesome!
